core/templates/domain/exploration/recorded-voiceovers.model.ts

```
@@ -46,12 +46,12 @@ export class RecordedVoiceovers {
     return Object.keys(this.voiceoversMapping);
   }
 
-  getBindableVoiceovers(contentId: string): BindableVoiceovers {
-    return this.voiceoversMapping[contentId];
+  getBindableVoiceovers(contentId: string | null): BindableVoiceovers {
+    return this.voiceoversMapping[String(contentId)];
```

Why these changes?

---

https://github.com/oppia/oppia/blob/8af8a439990f4ea692d9def3828fddc5b43747b0/core/templates/domain/exploration/subtitled-html.model.ts#L21

The contentId is of type `string | null` hence the type of parameter is changed. Also strict typescript checks don't allow indexing in array with a null value. Therefore, contentId is passed as a String during indexing.

---

I think `contentId == null` is unexpected and I would rather throw an error if that happens.

---

This function is not used anywhere, so removed it.